.github/actions/build-jtreg/action.yml

```
@@ -53,7 +53,14 @@ runs:
     - name: 'Build JTReg'
       run: |
         # Build JTReg and move files to the proper locations
-        bash make/build.sh --jdk "$JAVA_HOME_17_X64"
+        retry=0
+        while ! bash make/build.sh --jdk "$JAVA_HOME_17_X64"; do
+          retry=$[$retry+1]
+          if [ $retry -gt 20 ];then
```

20 retries seems excessive. There is likely a persistent problem if it retries that many times.

```suggestion
          if [ $retry -gt 3 ];then
```

---

Done. I have also added a sleep 30.